### PR TITLE
Improve site organization + visibility: Features page

### LINKED ISSUE #2388: Improve site organization + visibility: Features page
As part of https://github.com/meilisearch/documentation/issues/2380, we can delete the "[Features](https://www.meilisearch.com/docs/learn/what_is_meilisearch/features)" page and move the list to `/learn/what_is_meilisearch/overview.mdx` under "Features"

### PR DESCRIPTION
**Pull request by maryamsulemani97**
closes #2388

Removes the features page and moves content to `learn/what_is_meilisearch/overview.mdx`. 

### For reviewers
Is the list of features on `learn/what_is_meilisearch/overview.mdx` too long?

## Redirects

Added in https://github.com/meilisearch/meilisearch.com/pull/121

### CODE REVIEW

**Review by guimachiavelli — CHANGES_REQUESTED** [review]
Looks good, but there are two things we can improve:

- If I recall correctly, our style guides establish line items should not end in punctuation marks
- The text in the older bits is not great. Might be out of scope for the linked issue, but if you have the time it wouldn't hurt to polish them a bit. You might want to check out the meilisearch/meilisearch readme—there's a similar list there and I think we reviewed it more or less recently.

File: learn/what_is_meilisearch/overview.mdx [hunk 1]
```diff
@@ -17,18 +17,24 @@ _Meilisearch helps the Rust community find crates on [crates.meilisearch.com](ht
...
-- [Search as you type](/learn/what_is_meilisearch/features#search-as-you-type): Results are updated on each keystroke. To make this possible, we use [prefix-search](/learn/advanced/prefix#prefix-search).
-- [Typo tolerance](/learn/what_is_meilisearch/features#typo-tolerant): Understands typos and misspellings.
-- [Tokenization](/learn/advanced/tokenization) in **English**, **Chinese**, and **all languages that use space as a word divider**.
-- **Return the whole document**: The entire document is returned upon search.
+- **Blazing fast**: Answers in less than 50 milliseconds
+- **Search as you type**: Results are updated on each keystroke using [prefix-search](/learn/advanced/prefix#prefix-search)
+- [Typo tolerance](/learn/configuration/typo_tolerance): Get relevant matches even when queries contain typos and misspellings
+- [Comprehensive language support](/learn/what_is_meilisearch/language): Optimized support for **Chinese, Japanese, Hebrew, and languages using the Latin alphabet**
+- **Returns the whole document**: The entire document is returned upon search
 - **Highly customizable search and indexing**:
```
Comment (guimachiavelli) [inline learn/what_is_meilisearch/overview.mdx:25]: Huh, just noticed we don't have anything here, which clashes hard against every other feature having some descriptive text. We should add something so it's in line with the other items.

**Review by guimachiavelli — CHANGES_REQUESTED** [review]
I think we're almost done, but missing a certain consistency/uniformity between list items. The majority of descriptions begins with a verb—configure, manage, filter. Is it possible to have all following the same pattern?

**Review by dichotommy — APPROVED** [review]
LGTM!

Gui is right of course about the phrasal consistency—would probably sound better if each feature bullet started with a verb—but does not seem a priority to do that right now.

For the redirect, I'm thinking straight to https://www.meilisearch.com/ . Sounds good?

**Review by guimachiavelli — APPROVED** [review]

**Pull request merged**